validation-test/compiler_crashers_2_fixed/github_issue_60932.swift

```diff
@@ -0,0 +1,6 @@
+// RUN: not %target-typecheck-verify-swift
+
+// https://github.com/apple/swift/issues/60932
+
+// expected-error@+1 {{expected precedence group name after ':' in operator declaration}}
+prefix operator ++: // expected-error {{only infix operators may declare a precedence}} {{19-20=}}
```

Comment:
This crasher is fairly simple, and we know the reason it crashes: should we add a proper test case instead?